docs/docs/widgets/vertical-divider.md

```diff
@@ -4,58 +4,59 @@ title: Vertical Divider
 
 ## General
 ### Tooltip
 
-A Tooltip is often used to specify extra information about something when the user hovers the mouse pointer over the widget.
+A Tooltip is often used to specify extra information about something when the user hovers the mouse pointer over the component.
 
-Under the <b>General</b> accordion, you can set the value in the string format. Now hovering over the widget will display the string as the tooltip.
+Under the <b>General</b> accordion, you can set the value in the string format. Now hovering over the component will display the string as the tooltip.
 
 </div>
 
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{paddingTop:'24px'}}>
 
-## Layout
+## Devices
 
-|  <div style={{ width:"100px"}}> Layout </div> |  <div style={{ width:"100px"}}> Description </div> |  <div style={{ width:"135px"}}> Expected Value </div>|
+|  <div style={{ width:"100px"}}> Devices </div> |  <div style={{ width:"100px"}}> Description </div> |  <div style={{ width:"135px"}}> Expected Value </div> |
 |:----- |:---------  |:------------- |
-| Show on desktop | Toggle on or off to display desktop view. | You can programmatically determining the value by clicking on `Fx` to set the value `{{true}}` or `{{false}}` |
-| Show on mobile  | Toggle on or off to display mobile view.  | You can programmatically determining the value by clicking on `Fx` to set the value `{{true}}` or `{{false}}` |
+| Show on desktop | Toggle to control whether the component is displayed on desktop devices. | Accepts Boolean values: `true` (visible) or `false` (hidden). You can programmatically determine the value using `Fx`. |
+| Show on mobile  | Toggle to control whether the component is displayed on mobile devices. | Accepts Boolean values: `true` (visible) or `false` (hidden). You can programmatically determine the value using `Fx`. |
 
 </div>
 
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
 
-## Styles
+<div style={{paddingTop:'24px'}}>
 
-| <div style={{ width:"100px"}}> Properties </div> | <div style={{ width:"100px"}}> Description </div> |
-| ----------- | ----------- |
-| Divider Color |  It is used to set the color of the divider. Use hex code to set the background color. |
-| Visibility |  This property is used to set the visibility of the divider. The property accepts Boolean value. |
+## Styles
 
+| <div style={{ width:"120px"}}> Property </div> | <div style={{ width:"150px"}}> Description </div> | <div style={{ width:"180px"}}> Configuration Options </div> |
+| ----------- | ----------- | ----------- |
+| Divider Color | Sets the color of the divider. Use hex code or color names. | Accepts any valid color value like hex (`#000000`), RGB, or predefined color names. |
+| Visibility | Controls whether the divider is visible. Accepts Boolean values. | `true` (visible), `false` (hidden) |
+| Box Shadow | Adds shadow around the divider to create depth. | Accepts `box-shadow` values like `none`, `2px 4px 6px rgba(0, 0, 0, 0.1)` |
```

Comment:
Row: Box Shadow, Column: Configuration Options
box-shadow should be a normal text and should not be enclosed in single ticks